tegra/tegra.c

```diff
@@ -156,6 +156,9 @@ int drm_tegra_bo_new(struct drm_tegra_bo **bop, struct drm_tegra *drm,
 	if (!drm || size == 0 || !bop)
 		return -EINVAL;
 
+	if (flags & ~DRM_TEGRA_GEM_FLAGS)
```

Comment:
I just noticed that this is incorrect, DRM_TEGRA_GEM_FLAGS have nothing to do with the BO creation flags.